cosmic-applet-power/data/com.system76.CosmicAppletPower.desktop

```diff
@@ -1,6 +1,5 @@
 [Desktop Entry]
-Name=Cosmic Applet Power
-Comment=Applet for Cosmic Panel
+Name=Power
```

Comment:
So we have "Power and Battery" and "Power"? That seems a bit confusing.

Reply:
I set it to the name from #394, but maybe we could set the "Power and Battery" applet to be just "Battery"? @WatchMkr

Reply:
Please use "User Session" instead of "Power".